build.gradle

```diff
@@ -1,3 +1,11 @@
+// Build Script Classpath
+buildscript {
+    ext {
+        springBootVersion = '3.1.2'
```

Comment:
Might you be able to bring this up further, at least to 3.1.7?  (At that point, Spring Boot has upgraded logback past vulnerable versions.)

Reply:
Aha - yes I will do that. Maybe that means we could also upgrade Spring Boot in TCL and remove the [explicit logback dependency](https://github.com/DataBiosphere/terra-common-lib/pull/127).

Reply:
Great idea!

Reply:
https://github.com/DataBiosphere/terra-common-lib/pull/130

Reply:
Bumped Janitor to 3.1.7 (3.2.1 was giving me issues). Will follow up with TCL after.